routers/authRouter.js

```diff
@@ -0,0 +1,43 @@
+const express = require('express');
+const router = express.Router();
+const db = require('../server/helpers/authHelpers');
+const bcrypt = require('bcryptjs');
+const { newToken } = require('../server/MiddleWare/middleware')
+
+router.post('/signup', (req, res) => {
+  const user = req.body;
+  const hashedPass = bcrypt.hashSync(user.password, 12)
+  user.password = hashedPass;
+  db.insertUser(user)
```

Comment:
Your Auth looks good Michael 👍 well done using your helpers to break the problem down into smaller steps.